src/relational.cpp

```diff
@@ -214,11 +214,16 @@ external_pointer<T> make_external_prot(const string &rclass, SEXP prot, ARGS &&.
 	return make_external_prot<RelationWrapper>("duckdb_relation", prot, res);
 }
 
-[[cpp11::register]] SEXP rapi_rel_order(duckdb::rel_extptr_t rel, list orders) {
+[[cpp11::register]] SEXP rapi_rel_order(duckdb::rel_extptr_t rel, list orders, r_vector<r_bool> ascending) {
 	vector<OrderByNode> res_orders;
 
+	OrderType order_type;
+	size_t i = 0;
+
 	for (expr_extptr_t expr : orders) {
```

Comment:
@Antonov548: What's the idiomatic C++ way to iterate over two "parallel" vectors of the same length, `orders` and `ascending` ?

Reply:
As I know there is no specific function in STL to do this. Usually it's range based for loop with `:` for single iterable objects and index based loop for multiple vectors.

Reply:
Can we use `zip_view()` or similar?

https://codereview.stackexchange.com/q/283638/10691